en/docs/administer/configuring-a-jdbc-user-store.md

```diff
@@ -166,14 +177,12 @@ change the `         [user_store]        ` section in the
     products.
 
 3.  Add the `           PasswordHashMethod          ` property to the
-    `           UserStoreManager          ` configuration for
-    `           JDBCUserStoreManager          ` . For example:
+    `           UserStoreManager          ` configuration by adding configuration
+    to `deployment.toml`, For example:
```

Comment:
...to the `UserStoreManager` configuration in the `deployment.toml` file. For example:"